index.js

```diff
@@ -116,7 +116,9 @@ const getJobsHandler = (jobPath) => {
     console.log({
       accountNumber: functionRequest.params.accountNumber,
       url: url,
-      method: 'GET'
+      method: 'GET',
+      referer: functionRequest.get('Referrer'),
```

Comment:
this could be `referrer` or `referer` as the spec had a misspelling. you might consider using both if you see any empty ones.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Reply:
I believe in express, they take care of that for you.

>The Referrer and Referer fields are interchangeable.

http://expressjs.com/en/4x/api.html